<?xml version="1.0"?>
<launch>

  <arg name="run_darknet" default="true" />
  <arg name="camera_name" default="camera" />

  <group ns="$(arg camera_name)" >
    <node if="$(arg run_darknet)" pkg="yolo_classification" type="yolo_classification" name="yolo_classification" output="screen" >
      <param name="darknet_cfg_file" value="$(find darknet_ros)/cfg/yolov3.cfg" />
      <param name="darknet_weights_file" value="$(find darknet_ros)/yolov3.weights" />
    </node>

    <node pkg="yolo_classification" type="synced_yolo_data" name="synced_yolo_data" output="screen" />
  </group>

</launch>
